src/OrleansLoggerProvider/LegacyOrleansLoggerProvider.csproj

```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <PackageId>Microsoft.Orleans.LegacyOrleansLoggerProvider</PackageId>
+    <Title>Microsoft Legacy Orleans Logger Provider</Title>
+    <Description>Library of LegacyOrleansLoggerProvider.</Description>
+    <PackageTags>$(PackageTags) Orleans</PackageTags>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <AssemblyName>LegacyOrleansLoggerProvider</AssemblyName>
+    <RootNamespace>Orleans.Extensions.Logging</RootNamespace>
+    <PackageTargetFallback Condition=" '$(BuildFlavor)' == 'Legacy' ">$(PackageTargetFallback);portable-net45+win8+wpa81+wp8</PackageTargetFallback>
```

Comment:
I don't think we need ```PackageTargetFallback``` here, since we're not referencing any NuGets that would require it.

Reply:
cannot build without this line. may need more work here. can we address this in another PR. and get the bigger part out?